packages/actor-init-sparql/config/sets/sparql-queryoperators.json

```diff
@@ -2,7 +2,6 @@
   "@context": [
     "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/actor-init-sparql/^1.0.0/components/context.jsonld",
     "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/runner/^1.0.0/components/context.jsonld",
-
```

Comment:
Please undo this line removal. This is here to logically separate similar packages.